python/tests/fixtures/chars_sample_athena.csv

```diff
@@ -0,0 +1,71 @@
+"pin","year","class","char_yrblt","char_bldg_sf","char_land_sf","char_beds","char_rooms","char_fbath","char_hbath","char_frpl","char_type_resd","char_cnst_qlty","char_apts","char_tp_dsgn","char_attic_fnsh","char_gar1_att","char_gar1_area","char_gar1_size","char_gar1_cnst","char_attic_type","char_bsmt","char_ext_wall","char_heat","char_repair_cnd","char_bsmt_fin","char_roof_cnst","char_use","char_age","char_site","char_ncu","char_renovation","char_porch","char_air","char_tp_plan"
```

Comment:
I wish we could just share these fixtures with the R package, but the R package needs the fixtures to be saved as RData objects, and I feel like it would be kind of weird to bundle that in the Python package. Down to give it a try if you have a strong preference, though.

Reply:
**suggestion:** IMO it's worth it to create a shared directory for these fixtures that the R scripts use to instantiate the `.rda` files and the Python package uses for fixtures. The dir would contain this data, saved as CSV and created via SQL (from the existing R `data-raw` scripts).